Jenkinsfile

```diff
@@ -63,13 +63,13 @@ lock(resource: "Lisk-Core-Nodes", inversePrecedence: true) {
 		parallel(
 			"Build cached dependencies" : {
 				node('master-01'){
-					sh '''#!/bin/bash -xe
-					if [ $JENKINS_PROFILE == "jenkins-extensive" ]; then
-						rm -Rf "$WORKSPACE/node_modules/"
+					sh """
```

Comment:
Please, if you think this is better, change all the cases along the file.

Reply:
The double quotes are not better, they mean groovy will interpolate the variables and not the shell. Thus I switched to double quotes only where that behaviour was desired.